google-cloud-core/pom.xml

```diff
@@ -111,7 +111,7 @@
     <dependency>
       <groupId>com.google.api</groupId>
       <artifactId>gax</artifactId>
-      <version>0.0.25</version>
+      <version>0.0.28-SNAPSHOT</version>
```

Comment:
Makes sense. I'll change this to `0.0.28` (without the `-SNAPSHOT`). If we don't update the version, the build still doesn't compile.